spree_billing_sisow.gemspec

```diff
@@ -2,25 +2,24 @@
 Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
   s.name        = 'spree_billing_sisow'
-  s.version     = '0.5.0'
+  s.version     = '0.5.1'
   s.summary     = 'Spree billing integration for Sisow payment provider'
   s.description = 'Spree billing integration for Sisow iDeal/Bancontact/Sofort/Paypal payments'
   s.required_ruby_version = '>= 1.9.3'
 
-  s.authors   = ['Sjors Baltus', 'Bèr Kessels']
+  s.authors   = ['Sjors Baltus', 'Bèr Kessels', 'Vernon de Goede']
   s.email     = ['gems@berk.es']
-  s.homepage  = 'http://github.com/berkes/spree_billing_sisow'
+  s.homepage  = 'https://github.com/vernondegoede/spree_billing_sisow'
```

Comment:
This would mean that your repo becomes the canonical upstream. Is that intentional?